### PR TITLE
tweak connect routes

### PR DESCRIPTION
**Pull request by shamilovtim**
- the connect routes were renamed to be OIDC standard
- there were validations that were part of the OIDC spec (probably havent done all of them though)
- updated comments

### CODE REVIEW

**Review by LiranCohen — APPROVED** [review]
🙇

File: src/http-api.ts [hunk 1]
```diff
@@ -376,19 +376,46 @@ export class HttpApi {
...
+      // Validate that `request_uri` was NOT provided
+      if (req.body?.request?.request_uri) {
+        return res.status(400).json({
+          ok: false,
+          status: {
+            code: 400,
+            message: "Bad Request: 'request_uri' parameter is not allowed in PAR",
+          },
+        });
+      }
```
Comment (thehenrytsai) [inline src/http-api.ts:409]: Sanity: are the new code covered by tests? If not, please makes sure we have 100% new code coverage.

Comment (shamilovtim) [reply src/http-api.ts:409]: > please makes sure we have 100% new code coverage

why? these are harmless validations not on the golden path.

Comment (thehenrytsai) [reply src/http-api.ts:409]: Hey @shamilovtim, are you suggesting that we should only test golden paths? Harmless validations still should be tested no? It'll take probably take longer time arguing about it than just adding tests to it when it is easy to do so in JS/TS.

In the spirit of this week's deadline, you are good to resolve this and merge, but I've filed an issue #149 and assigned to you as a fast-follow.

Comment (shamilovtim) [reply src/http-api.ts:409]: > are you suggesting that we should only test golden paths? Harmless validations still should be tested no?

@thehenrytsai  Yes, I'm saying that all code does not need to be tested. I don't think that's a controversial statement. 100% coverage is actually a far more controversial thing to suggest.

**Pull request merged**